test/acceptance/to-js-test.js

```diff
@@ -56,4 +56,29 @@ describe('acceptance - to-js', function() {
       }).catch(done);
     });
   });
+
+  it('handles option --merge', function(done) {
+    fs.copySync('test/fixtures/locales', 'tmp/locales', { clobber: true });
+    let ps = spawn(process.execPath, [
+      'bin/ember-i18n-csv.js',
+      'to-js',
+      '--csv-path=test/fixtures/i18n-merge.csv',
+      '--locales-path=tmp/locales',
+      '--merge'
+    ]);
+
+    let out = '';
+    let err = '';
+    ps.stdout.on('data', buffer => out += buffer);
+    ps.stderr.on('data', buffer => err += buffer);
+
+    ps.on('exit', () => {
+      expect(out).to.equal('');
+      expect(err).to.equal('');
+      areDirsEqual('tmp/locales', 'test/fixtures/locales').then(areSame => {
```

Comment:
Done again. But i'm not exactly sure why the test wasn't failing before. I've never used this test suite before so i'm kinda shooting in the dark here.